wp-content/themes/citylimits/functions.php

```diff
@@ -612,3 +649,139 @@ function tribe_remove_customizer_css(){
+ */
+remove_action( 'wp_enqueue_scripts', 'largo_enqueue_child_theme_css' );
+
+/**
+ * filter search results: remove old events
+ */
+function cl_pre_get_posts($query) {
+	if ( !is_admin() && $query->is_main_query() && $query->is_search ) {
+		$meta_query = $query->get('meta_query');
+		$additional_query = array(
+			'relation' => 'OR',
+			array(
+				'key' => '_EventStartDate',
+				'value' => date("Y-m-d H:i:s"),
+				'compare' => '>=',
+				'type' => 'DATETIME'
+			) ,
+			array(
+				'key' => '_EventStartDate',
+				'compare' => 'NOT EXISTS'
+			)
+		);
+		if ( is_array( $meta_query ) ) {
+			$meta_query[] = $additional_query;
+		} else {
+			$meta_query = $additional_query;
+		}
+
+		$query->set('meta_query', $meta_query);
+	}
+	return $query;
+}
+
+add_action( 'pre_get_posts', 'cl_pre_get_posts' );
+
+
+/**
+ * JETPACK RELATED POSTS
+ */
+ 
+//remove JP Related Posts from default location so we can move it elsewhere
+function jetpackme_remove_rp() {
+    if ( class_exists( 'Jetpack_RelatedPosts' ) ) {
+        $jprp = Jetpack_RelatedPosts::init();
+        $callback = array( $jprp, 'filter_add_target_to_dom' );
+        remove_filter( 'the_content', $callback, 40 );
+    }
+}
+add_filter( 'wp', 'jetpackme_remove_rp', 20 );
+
+
+// limit results to last 3 years
+function jetpackme_related_posts_past_3years_only( $date_range ) {
+	$date_range = array(
+		'from' => strtotime( '-3 years' ),
+		'to' => time(),
+	);
+	return $date_range;
+}
+add_filter( 'jetpack_relatedposts_filter_date_range', 'jetpackme_related_posts_past_3years_only' );
+
+
+//eliminate some categories
+function jetpackme_filter_exclude_category( $filters ) {
+	$filters[] = array( 'not' =>
+		array( 'term' => array( 'category.slug' => 'community-wire' ) )
+	);
+	$filters[] = array( 'not' =>
+		array( 'term' => array( 'term' => 'uncategorized' ) )
+	);
+	return $filters;
+}
+add_filter( 'jetpack_relatedposts_filter_filters', 'jetpackme_filter_exclude_category' );
+
+//dequeue jetpack css so we can override it
+add_filter( 'jetpack_implode_frontend_css', '__return_false', 99 );
+
+/**
+ * newsletter subscribe forms
+ */
+
+function citylimits_newsletter_form_interstitial() {
+	get_template_part( 'partials/newsletter-signup', 'maincolumn' );
+}
+add_action( 'largo_before_sticky_posts', 'citylimits_newsletter_form_interstitial', 11 );
+add_action( 'largo_category_after_primary_featured_post', 'citylimits_newsletter_form_interstitial', 11 );
+add_action( 'largo_series_before_stories', 'citylimits_newsletter_form_interstitial', 11 );
+add_action( 'largo_archive_before_stories', 'citylimits_newsletter_form_interstitial', 11 );
+
+function citylimits_newsletter_form_footer() {
+	get_template_part( 'partials/newsletter-signup', 'footer' );
+}
+add_action( 'largo_before_footer', 'citylimits_newsletter_form_footer', 11 );
+
+function citylimits_newsletter_form_popover() {
+	get_template_part( 'partials/newsletter-signup', 'popover' );
+}
+if (!$_COOKIE['newsletter_modal_snooze']) {
```

Comment:
This is probably not going to work in production; Flywheel uses caching. It may be better to instead enqueue a bit of JS in the newsletter partial that checks the cookie value, then hides or shows the footer popover.

Reply:
if you're staying with the cookie check, please:

```diff
- if (!$_COOKIE['newsletter_modal_snooze']) {
+ if ( isset( $_COOKIE['newsletter_modal_snooze'] ) && true == $_COOKIE['newsletter_modal_snooze'] ) {
111